sys/net/network_layer/fib/fib.c

```diff
@@ -547,7 +548,7 @@ int fib_get_destination_set(fib_table_t *table, uint8_t *prefix,
 
     for (size_t i = 0; i < table->size; ++i) {
         if ((table->data.entries[i].global != NULL) &&
-            (universal_address_compare_prefix(table->data.entries[i].global, prefix, prefix_size<<3) >= 0)) {
+            (universal_address_compare_prefix(table->data.entries[i].global, prefix, prefix_size<<3) > -ENOENT)) {
```

Comment:
If we want to state that a positive return means match, `>0`could also work? But the current solution seems a bit... un-concise. sorry for he nitpicking ^^